### DISCUSSION TITLE
Support for local M3U playlist http streams

### DISCUSSION THREAD

**Discussion by dazWiLLiE** [music-assistant/support#408] [Feature requests and ideas]
Hi, trying out MA and it works great.

One question though, will local m3u with http streams be supported at some point? I have a premium DI.fm account and get a pls/m3u from them with premium streaming-links. MA finds the m3u, but its empty.

**Comment by erkr** [comment]
I moved this request to the discussions, thanks for filing.

**Comment by dazWiLLiE** [comment]
Any updates on this? You said on discord that it probably was an easy fix :)

**Comment by meichthys** [comment]
This would be great since it should allow for adding playlists from the great radio-browser.info resource

**Comment by OzGav** [comment]
Relates to v1. Radio browser is available
